solar_system.rb

```diff
@@ -0,0 +1,31 @@
+# SolarSystem, which is responsible for keeping
+# track of a collection of instances of Planet.
+require_relative 'planet'
+
+class SolarSystem
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+  end
+
+  attr_reader :star_name, :planets
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def list_planets
+    puts "Planets orbiting #{@star_name}"
```

Comment:
Avoid putting a `puts` in a method that returns a value.  You don't ALWAYS want to print immediately.